client/ayon_core/hosts/maya/api/lib_renderproducts.py

```diff
@@ -1133,9 +1133,26 @@ def get_render_products(self):
             aovs = list(set(aovs) - set(ref_aovs))
 
         products = []
+        # global_aov_enabled = bool(
+        #     self._get_attr("redshiftOptions.aovGlobalEnableMode")
+        # )
+        global_aov_enabled = bool(
+            cmds.getAttr("redshiftOptions.aovGlobalEnableMode"))
```

Comment:
@BigRoy, pretty strange that `self._get_attr` doesn't work for this case. 
Not sure if the bool convert is the issue.

Reply:
I understand why! :) `self._get_attr` likely calls the get attribute with `asString=True` and thus returns the 'string' result of the attribute and not the numeric one?

Anyway, please do still use `_get_attr` since it actually considers renderlayer overrides which it must (since it could be disabled just in one layer (we do that all the time).

We might want to expose the `as_string` argument to `_get_attr` and for this call have it be `_get_attr(attr, as_string=False)`?